plugins/qcheck-stm/src/stm_of_ir.ml

```diff
@@ -910,22 +910,41 @@ let agree_prop =
       check_init_state ();
       STMTests.agree_prop cs]
 
-let stm include_ config ir =
+let prepend_include_in_module name lident structure =
+  let open Ast_helper in
+  let name = noloc (Some name)
+  and expr =
+    pmod_structure
+    @@ ((Mod.ident lident |> Incl.mk |> pstr_include) :: structure)
+  in
+  [ pstr_module @@ module_binding ~name ~expr ]
+
+let qcheck config =
```

Comment:
Maybe rename it (`qcheck_module_wrapper`?)? `qcheck` seems too ambiguous in a QCheck-STM plugin.